### PR TITLE
Add colors

### PR DESCRIPTION
**Pull request by e6**
Added `colors` folder with `colors.txt`, a list of all colors. 

These are considered as OG usernames.

### CODE REVIEW

**Comment by l3nnartt** [comment]
I guess it would be better to put Giselle and the colors in separate PRs

**Review by Kalsie — CHANGES_REQUESTED** [review]
Kalsie shouldn't be removed especially after it's already received approval.

**Comment by Dqnns** [comment]
Kalsie is a common first name in the USA, I don’t see why it should get
reverted.

Kalsie ***@***.***> schrieb am Mo. 13. März 2023 um 05:17:

File: src/names/names.txt [hunk 1]
```diff
@@ -1040,6 +1040,7 @@ Ginger
 Giovanni
 Gisbert
 Gisela
+Giselle
```
Comment (l3nnartt) [inline src/names/names.txt:1043]: You should still put Giselle in a separate pr

**Review by l3nnartt — CHANGES_REQUESTED** [review]

**Comment by Dqnns** [comment]
run cleanup.py

**Pull request closed without merging**